src/index/CompressedRelation.cpp

```diff
@@ -1714,11 +1790,15 @@ auto CompressedRelationReader::getScanConfig(
   // will throw an assertion.
   auto [graphColumnIndex,
         deleteGraphColumn] = [&]() -> std::pair<ColumnIndex, bool> {
-    if (!scanSpec.graphsToFilter().has_value()) {
-      // No filtering required, these are dummy values that are ignored by the
-      // filtering logic.
-      return {0, false};
-    }
+    // TODO<joka921> figure out whether this is the correct fix.
+    /*
+if (!scanSpec.graphsToFilter().has_value()) {
+// No filtering required, these are dummy values that are ignored by the
+// filtering logic.
+// TODO<joka921> Those are not at all dummies...
+return {0, false};
+}
```

Comment:
This code is honestly confusing

Reply:
This code was not needed and not correct, I have deleted it.